node/src/gql/middleware.ts

```diff
@@ -27,7 +27,7 @@ async function refreshToken(req: Request, next: any) {
   // eslint-disable-line  @typescript-eslint/no-explicit-any
   try {
     logger.debug('refreshing token');
-    const token = req.cookies.refreshToken;
+    const token = req.cookies[process.env.REFRESH_TOKEN_NAME];
```

Comment:
this might be undefined if envvar is not set so perhaps just log the value to debug later if needed

Reply:
[16777a0](https://github.com/mentorpal/mentor-graphql/pull/98/commits/16777a035ce42a029e9048b4ec54982e0551fb30)